src/pages/storage/StorageBucketLink.tsx

```diff
@@ -0,0 +1,23 @@
+import type { FC } from "react";
+import { Link } from "react-router-dom";
+import ItemName from "components/ItemName";
+import { LxdStorageBucket } from "types/storage";
+
+interface Props {
+  bucket: LxdStorageBucket;
+}
+
+const StorageBucketLink: FC<Props> = ({ bucket }) => {
+  return (
+    <Link
+      to={`/ui/project/${bucket.project}/storage/pool/${bucket.pool}/bucket/${bucket.name}`}
```

Comment:
This should use the shared function to generate the link from the bucket object.